README.md

```diff
@@ -1,53 +1,75 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# HTML CSS Javascript Capstone Project
+# HTML CSS JS Capstone Project
 
-> One paragraph statement about the project.
+> Hello, My name is Zain Sadaqat and this is my HTML CSS JS Capstone project. Kindly take a look by visiting the link given below. :)
 
-![screenshot](./app_screenshot.png)
+## Desktop View
 
-Additional description about the project and its features.
+![DesktopView](media/images/Desktop-View.png)
 
-## Built With
+## Mobile View
 
-- HTML5
-- CSS
-- Javascript
+![MobileView](media/images/mobile-view.png)
 
-## Live Demo
+> This project is all about a webinar which is related to Cyber Security and going to held in 2021 and due to COVID it couldn't be held in last years. There're some well-known speakers who will be addressing in this webinar and going to talk about future trend and need of cyber security.
 
-[Live Demo Link](https://livedemo.com)
+## Live Project Link
 
+> [Live Project Link:](https://zainsadaqat.github.io/html-css-js-capstone/)
```

Comment:
- [ ] The live link demo you added doesn't open the live working properly, please fix this. see the image below
![navb](https://user-images.githubusercontent.com/55635977/128437615-ecad8f23-e971-46bb-a479-cd057e294566.png)

- [ ] Attach the loom video here

Reply:
This is because I hosted the master branch on Github pages which is currently empty and the master branch is yet to be merged with the html-css-js-capstone branch which is my complete project.  

Let me host html-css-js-branch on github pages and it'll start working properly.

Reply:
# Fix Live Project Link's Loom Video
https://www.loom.com/share/fe524b0c6cf34478942365968712dbc9